kubernetes/kites/net-test-dev_ds.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: net-test-dev-ds
+  labels:
+    app: net-test-dev-ds
+spec:
+  selector:
+    matchLabels:
+      app: net-test-dev
+  template:
+    metadata:
+      labels:
+        app: net-test-dev
+    spec:
+      containers:
+        - name: net-test-dev-server
```

Comment:
I'd call this `iperf3-server`

Reply:
@ferrarimarco I had asked to change from iperf3 to net-test-dev
iperf3 is only a small part (may be in the future it can also be removed if we do not need this specific tool)

Reply:
My fear is that it encourages putting all the stuff in a single container

Reply:
it is good to remind this general design principle, but in our case we are not deploying a "normal" distributed application which needs to be split in containers as "small" as possible

we need a container probe that needs to run our performance tests, so we will have to put in the container everything we need to run the test

Reply:
/shrug